api/v01/entities/vrp_result.rb

```diff
@@ -26,6 +26,7 @@ class VrpResultSolutionRouteActivityDetailTimewindows < Grape::Entity
 
     class VrpResultDetailQuantities < Grape::Entity
       expose :unit, documentation: { type: String, desc: '' }
+      expose :label, documentation: { type: String, desc: '' }
```

Comment:
what is unit? Isn't that the label of the unit? Or is `unit` the id? :thinking:

Reply:
That is a legacy typo from me :D `unit` should be `id`